health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/timelinePopUpWrapper.js

```
@@ -0,0 +1,780 @@
+                showConnector: true
+            }));
+            setTimelineSteps(steps);
+        }
+    }, []);
+
+    return (
+        <PopUp
+            onClose={onClose}
+            heading={t(heading)}
+            onOverlayClick={onClose}
+            children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)